Makefile

```diff
@@ -15,10 +15,10 @@
 .PHONY: all rbdplugin cephfsplugin
 
 RBD_IMAGE_NAME=$(if $(ENV_RBD_IMAGE_NAME),$(ENV_RBD_IMAGE_NAME),quay.io/cephcsi/rbdplugin)
-RBD_IMAGE_VERSION=$(if $(ENV_RBD_IMAGE_VERSION),$(ENV_RBD_IMAGE_VERSION),v0.3.0)
+RBD_IMAGE_VERSION=$(if $(ENV_RBD_IMAGE_VERSION),$(ENV_RBD_IMAGE_VERSION),v1.0.0)
 
 CEPHFS_IMAGE_NAME=$(if $(ENV_CEPHFS_IMAGE_NAME),$(ENV_CEPHFS_IMAGE_NAME),quay.io/cephcsi/cephfsplugin)
-CEPHFS_IMAGE_VERSION=$(if $(ENV_CEPHFS_IMAGE_VERSION),$(ENV_CEPHFS_IMAGE_VERSION),v0.3.0)
+CEPHFS_IMAGE_VERSION=$(if $(ENV_CEPHFS_IMAGE_VERSION),$(ENV_CEPHFS_IMAGE_VERSION),v1.0.0)
```

Comment:
If we could refrain from tagging the cephfs plugin as 1.0 just yet and instead have this as let's say 0.9. I wanted to use this major version bump as an opportunity to make some compatibility-breaking changes (e.g. getting rid of the `provisionVolune` attribute), but @mcronce was faster with this PR than me ;) 

@rootfs ?

Reply:
@mcronce @gman0 what if we get a csi-v1.0 branch and commit the changes there?

Reply:
Sounds good!

Reply:
@mcronce @gman0 i created csi-v1.0 branch https://github.com/ceph/ceph-csi/tree/csi-v1.0

Reply:
@mcronce can you open an PR against csi-v1.0 branch? We'll change the Makefile as well to create container images using csi-v1.0 tag

Reply:
@rootfs Done - #105